app/shop.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+import dataclasses
+import datetime
+
+from app.customer import Customer
+
+
+@dataclasses.dataclass
+class Shop:
+    name: str
+    location: list
+    products: dict
+
+    def shop_print_receipt(self, customer: Customer) -> None:
+        milk_cost = self.products["milk"] \
+            * customer.product_cart["milk"]
+        bread_cost = self.products["bread"] \
+            * customer.product_cart["bread"]
+        butter_cost = self.products["butter"] \
+            * customer.product_cart["butter"]

```

Comment:
use get() method instead of parenthesis **everywhere** to avoid KeyError

Reply:
Done